docs/contributing/guidelines/workflow.md

```diff
@@ -277,3 +277,12 @@ The following labels summarize the scope of the pull request:
 - *scope: feature*.
 - *scope: new-target*.
 - *scope: refactor*.
+
+### Documentation contribution labels
+
+The [documentation repository](https://github.com/ARMmbed/mbed-os-5-docs) uses some of the same labels that the `mbed-os` repository does. In addition, it includes: 
+
+- *code dependency* - The feature or change this pull request describes has not yet merged into `master` in `mbed-os`. Even if all the required reviewers have reviewed and approved this pull request, it must wait to mege until its code does.
+- *wait to merge* - This pull request is still a work in progress and is not yet ready for full review. 
```

Comment:
I would say either "still in progress" or "is a work in progress". I don't think it needs both variants.
Also, I didn't know wait to merge refers to the review status; I assumed it was reviewed but not yet ready to merge for a reason other than code dependency.